api_tests/entities/merge.test.coffee

```diff
@@ -74,3 +74,40 @@ describe 'entities:merge', ->
     .catch undesiredErr(done)
 
     return
+
+  it 'should transfer claims', (done)->
```

Comment:
suggestion : `should transfer work claims to another work`

Reply:
it's testing for entities in general: `should transfer entity claims to the target entity`?